UNL-CSE-AMBASSADORS/VITA
# 124 dependent appointments

Created the idea of a DependentClient -- That is, our response to the situation where a client wants to sign up and bring their 4 kids to the appointment as well. So this enables a person to sign up themselves, and sign up their 4 dependents at the same time. What happens on the backend, is one appointment is created, one client is created, and then 4 DependentClients are created. 

Also sorry it doesn't look good @CollinsSpencer. Will let you restyle it when you do #126 ❤️ 


**TESTING**
* Go to the /signup page
* Hit the add dependent button -- Ensure that the two inputs / remove button gets added to the page
* Fill in all the information on the form and submit it
* Check the database and ensure that the DependentClient table has the correct information for what you entered
* Verify that the /management/documents download appointment schedule still works and contains the "Number of Returns" column.

**CHANGES TO PROD DB**
This script will need to be run against the PROD database prior to this being deployed in PROD. There is no impact of running this in prod before this PR is deployed.
```
CREATE TABLE DependentClient (
	dependentClientId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
	firstName VARCHAR(255) NOT NULL,
	lastName VARCHAR(255) NOT NULL,
	clientId INTEGER UNSIGNED NOT NULL,
	FOREIGN KEY(clientId) REFERENCES Client(clientId)
);
```
